src/components/map/Map.tsx

```diff
@@ -84,10 +82,15 @@ class LeafletMap extends React.Component<IMapProps> {
             this.props.mapStore!.setZoom(map.getZoom());
         });
 
-        this.reactionDisposer = reaction(() =>
-        [this.props.mapStore!.coordinates],
-                                         this.centerMap,
-            );
+        this.reactionDisposer = reaction(
+            () => this.props.mapStore!.coordinates,
+            this.centerMap,
+        );
+
+        this.reactionDisposer = reaction(
```

Comment:
These both need a separate disposer and they both need to be disposed of when component unmounts.
Now it's just replacing one disposer with the other and only disposing one of them afaik.

Reply:
Yeah, I had just copied the thing. Forgot about handling the reactionDisposers. Thanks for noticing!